pkg/datapath/linux/routing/routing.go

```diff
@@ -209,6 +209,15 @@ func Delete(ip net.IP, compat bool) error {
 		scopedLog.WithField(logfields.Rule, egress).Debug("Deleted egress rule")
 	}
 
+	// Mark IP as unreachable to avoid triggering rp_filter after endpoint deletion for new packets to pod IP
```

Comment:
Do the comments of this function now need updating?

Reply:
I moved the comment after the test, it feels clearer
What do you think?

Reply:
LGTM